### PR TITLE
Update credits

### PR DESCRIPTION
**Pull request by RockySouthpaw**
**Pull Request Description**

_Please include a general description of the changes made, why they were necessary, 
and any other general things to note._

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [ ] Have you built and tested NPWD in-game after the relevant change? kek

### CODE REVIEW

File: docs/npwd/Credits.md [hunk 1]
```diff
@@ -4,21 +4,22 @@ title: Contributers
...
-We appreciate all the support and help we get. Looking forward to making this list longer!
+We appreciate all the support and help we get. Below is a list of contributors to the resource with the [**Project Error Team**](https://github.com/orgs/project-error/people) removed from the list. We hope we continuously add to this list as time passes.
 
-- [**Oldarorn**](https://github.com/Oldarorn)
 - [**AvarianKnight**](https://github.com/AvarianKnight)
 - [**BombayV**](https://github.com/BombayV)
-- [**synterrr**](https://github.com/synterrr)
-- [**dayiskuan**](https://github.com/dayiskuan)
-- [**brezedc**](https://github.com/brezedc)
-- [**BTNGaming**](https://github.com/BTNGaming)
```
Comment (RockySouthpaw) [inline docs/npwd/Credits.md:15]: I removed/ didnt include contributors who made irrelevant comments like [licenses](https://github.com/project-error/npwd/commits?author=BTNGaming) and [print removal](https://github.com/project-error/npwd/commit/42830ecdace5647175df1a674df0fb30382cbcdc)

Comment (itschip) [reply docs/npwd/Credits.md:15]: Fair enough

**Comment by RockySouthpaw** [comment]
List is also alphabetical now.

**Comment by itschip** [comment]
By all means, merge this 😀  Do you also know how to deploy this? Or do you want me to do it?

**Comment by RockySouthpaw** [comment]
> By all means, merge this 😀 Do you also know how to deploy this? Or do you want me to do it?

kek i have no clue how deployment works but if you explain it i can do it as i can further update the docs and deploy as needed later

File: docs/npwd/Credits.md [hunk 2]
```diff
@@ -4,21 +4,22 @@ title: Contributers
 sidebar_label: Contributers
 ---
 
-We appreciate all the support and help we get. Looking forward to making this list longer!
+We appreciate all the support and help we get. Below is a list of contributors to the resource with the [**Project Error Team**](https://github.com/orgs/project-error/people) removed from the list. We hope we continuously add to this list as time passes.
```
Comment (AvarianKnight) [inline docs/npwd/Credits.md:7]: Would probably be better worded with "Below is a list of community contributors"

Comment (RockySouthpaw) [reply docs/npwd/Credits.md:7]: Will focus test this with the focus group.

**Pull request merged**